independent-publisher-connectors/Coupa/apiProperties.json

```diff
@@ -0,0 +1,115 @@
+                     "value":"code={code}&grant_type=authorization_code&redirect_uri={redirectUrl}&client_id={CoupaClientId}&client_secret={CoupaClientSecret}&scope={scope}"
+                  },
+                  "refreshBodyTemplate":{
+                     "value":"code={code}&grant_type=authorization_code&redirect_uri={redirectUrl}&client_id={CoupaClientId}&client_secret={CoupaClientSecret}&scope={scope}"
+                  }
+               }
+            }
+         },
+         "token:customHost":{
+            "type":"string",
+            "uiDefinition":{
+               "constraints":{
+                  "tabIndex":2,
+                  "required":"true"
+               },
+               "description":"Enter Coupa Instance name excluding coupahost.com",
+               "displayName":"Coupa Instance name",
+               "tooltip":"Coupa Instance name"
+            }
+         },
+         "token:scope":{
+            "type":"string",
+            "uiDefinition":{
+               "constraints":{
+                  "tabIndex":3,
+                  "required":"true"
+               },
+               "description":"Enter Coupa Oidc scopes with no commas",
+               "displayName":"Coupa Scope",
+               "tooltip":"Coupa Scope"
+            }
+         },
+         "token:CoupaClientId":{
+            "type":"string",
+            "uiDefinition":{
+               "constraints":{
+                  "tabIndex":4,
+                  "required":"true"
+               },
+               "description":"Enter Coupa Client Id",
+               "displayName":"Coupa Client Id",
+               "tooltip":"Coupa Client Id"
+            }
+         },
+         "token:CoupaClientSecret":{
+            "type":"securestring",
+            "uiDefinition":{
+               "constraints":{
+                  "tabIndex":5,
+                  "required":"true"
+               },
+               "description":"Enter Coupa Client Secret",
+               "displayName":"Coupa Client Secret",
+               "tooltip":"Coupa Client Secret"
+            }
+         }
+      },
+      "iconBrandColor":"#268ecd",
+      "capabilities":[
+         "actions"
+      ],
+      "policyTemplateInstances":[
+         {
+            "templateId":"dynamichosturl",
+            "title":"Set host for connection",
+            "parameters":{
+               "x-ms-apimTemplateParameter.urlTemplate":"https://@connectionParameters('token:customHost').coupahost.com"
+            }
+         },
+         {
+            "templateId":"setheader",
+            "title":"Accept",
+            "parameters":{
+               "x-ms-apimTemplateParameter.name":"Accept",
+               "x-ms-apimTemplateParameter.value":"application/json",
+               "x-ms-apimTemplateParameter.existsAction":"override",
+               "x-ms-apimTemplate-policySection":"Request"
+            }
+         }
+      ],
+      "publisher":"NovaGL",
+      "stackOwner":"NovaGL"

```

Comment:
Hi, thanks for that I was debating what it should have been.

Both these fields have been renamed to "Coupa Software"

Reply:
@NovaGL if you are the API owner then both should be the same with your company name. Otherwise your company name will be the "publisher" and the "stackOwner" will be the company that owns the API.
Maybe in your case it should be.
```
"publisher":"NovaGL",
"stackOwner":"Coupa Software"
```

Reply:
Thanks, this has been updated